### PR TITLE
Selection Race Case Bug

### PR DESCRIPTION
**Pull request by KyroVibe**
### Description
The selection inputs for the exporter had a race case that just happened to work for a while... until a recent update.

### CODE REVIEW

File: exporter/SynthesisFusionAddin/src/UI/ConfigCommand.py [hunk 1]
```diff
@@ -2109,22 +2148,17 @@ def notify(self, args):
             elif cmdInput.id == "wheel_select":
                 self.reset()
 
-                wheelSelect.isEnabled = False
-                wheelSelect.isVisible = False
+                # wheelSelect.isVisible = False
```
Comment (BrandonPacewic) [inline exporter/SynthesisFusionAddin/src/UI/ConfigCommand.py:2151]:
```suggestion
```

**Review by BrandonPacewic — COMMENTED** [review]
Looks like the `privacy/beare` workflow is failing for some reason.

**Review by BrandonPacewic — APPROVED** [review]
I think everything is working fine. I'm going to need to spend more time with the exporter at some point to get more comfortable with it.

**Comment by a-crowell** [comment]
It exported a CAD model into Synthesis successfully. Are there more steps to reproduce the bug?

**Pull request merged**